### PR TITLE
feat: add quarkus to java part

### PR DESCRIPTION
**Pull request by Alexsky347**
## What Changed

Added 4 new Quarkus skills (quarkus-patterns, quarkus-security, quarkus-tdd, quarkus-verification) and propagated them across the entire codebase, mirroring how the existing springboot-* skills are referenced:

Added origin: ECC to all 4 quarkus SKILL.md frontmatters
- Updated rules/java/ (testing, patterns, security) with quarkus skill references
- Updated manifests/install-modules.json with quarkus skill paths in framework-language and security modules
- Updated agent.yaml with 4 new quarkus skill entries
- Updated README.md and README.zh-CN.md directory trees and install examples
- Updated skills/configure-ecc/SKILL.md table (21->25 skills), category label, and cross-references
- Updated skills/prompt-optimizer/SKILL.md with Quarkus/Java tech stack row
- Updated .opencode/ agent files (java-reviewer.txt, java-build-resolver.txt) with dual Spring/Quarkus references
- Created translated quarkus skill copies and updated references in docs/zh-CN/, docs/tr/, docs/ja-JP/

## Why This Change

The quarkus-* skills were created on this branch but only existed as standalone files — they weren't registered in manifests, referenced in rules, listed in READMEs, or included in the installer/optimizer skills. This made them invisible to users browsing the catalog or running /configure-ecc. This PR ensures feature parity with springboot-* skills across all discovery surfaces.

## Testing Done

- [x] Manual testing completed
- [x] Automated tests pass locally (`node tests/run-all.js`)
- [x] Edge cases considered and tested

## Type of Change
- [ ] `fix:` Bug fix
- [x] `feat:` New feature
- [ ] `refactor:` Code refactoring
- [ ] `docs:` Documentation
- [ ] `test:` Tests
- [ ] `chore:` Maintenance/tooling
- [ ] `ci:` CI/CD changes

## Security & Quality Checklist
- [x] No secrets or API keys committed (ghp_, sk-, AKIA, xoxb, xoxp patterns checked)
- [x] JSON files validate cleanly
- [ ] Shell scripts pass shellcheck (if applicable)
- [ ] Pre-commit hooks pass locally (if configured)
- [x] No sensitive data exposed in logs or output
- [x] Follows conventional commits format

## Documentation
- [x] Updated relevant documentation
- [x] Added comments for complex logic
- [x] README updated (if needed)

---
## Summary by cubic
Adds first-class Quarkus support to the Java stack with automatic framework detection, Quarkus-aware agents, and updated coding standards. Updates docs, rules, manifests, and installer to surface Quarkus across the platform.

- **New Features**
  - Added `quarkus-patterns`, `quarkus-security`, `quarkus-tdd`, `quarkus-verification` with localized copies (EN, ja-JP, tr, zh-CN).
  - `java-reviewer` and `java-build-resolver`: auto-detect Spring Boot or Quarkus, show Framework output, link to matching skills; apply framework-specific checks.
  - `java-coding-standards` and `prompt-optimizer`: extended to Quarkus with CDI/Panache/reactive conventions and build-file detection; manifests/installer list Quarkus skills.

- **Bug Fixes**
  - `java-build-resolver`: added Gradle equivalents for Quarkus commands; split guidance into Maven/Gradle/Common.
  - Security: removed `unsafe-inline` from `script-src` in CSP; rate limiter now uses `getRemoteAddr()`; auth filter rejects missing/malformed `Authorization` headers.
  - Quarkus docs/tests: fixed compile/test issues (JSON marshalling, checked exceptions, missing mocks/fields, S3 test); clarified AssertJ vs JUnit; switched to `io.quarkus.platform:quarkus-camel-bom`.
  - Verification docs: corrected Mockito for Panache `persist()` with `doNothing()`; Turkish TDD example now expects `NullPointerException` for `Objects.requireNonNull`.

<sup>Written for commit 11cc20bbcfdb611ff1575a4b570dbc4bc1c2d85d. Summary will update on new commits.</sup>

## Summary by CodeRabbit

* **New Features**
  * Added Quarkus support with framework detection and framework-aware reviewer/resolver behavior
  * Introduced four Quarkus skills: patterns, security, TDD, and verification

* **Documentation**
  * Expanded Java guidance to cover both Spring Boot and Quarkus
  * Added extensive Quarkus docs across languages and updated installer/manifest listings

### CODE REVIEW

File: docs/zh-CN/skills/quarkus-patterns/SKILL.md [hunk 1]
```diff
@@ -0,0 +1,754 @@
+---
+name: quarkus-patterns
+description: Quarkus 3.x LTS architecture patterns with Camel for messaging, RESTful API design, CDI services, data access with Panache, and async processing. Use for Java Quarkus backend work with event-driven architectures.
+origin: ECC
+---
```
[the remark this answers is not in the document]
Comment (Alexsky347) [reply docs/zh-CN/skills/quarkus-patterns/SKILL.md:5]: 7945c3c979cd0b3302f2d76483a2c1c2fb90f3d2

File: skills/quarkus-patterns/SKILL.md [hunk 2]
```diff
@@ -0,0 +1,754 @@
...
+                    log.error(errorMsg);
+                    this.eventService.createErrorEvent(documentInfo, "FILE_UPLOAD_FAILED", errorMsg);
+                    throw new As2ServerProcessingException(errorMsg);
+                }
+                
+                this.eventService.createSuccessEvent(documentInfo, "PERSISTENCE_BLOB_EVENT_TYPE");
+                
+                BusinessRulesPayload payload = this.documentJobService.createDocumentAndJobEntities(
+                    documentInfo, originalFileName, structureIdPartner, 
+                    flowProfile, invoiceValidationResult.getDocumentHash());
```
[the remark this answers is not in the document]
Comment (Alexsky347) [reply skills/quarkus-patterns/SKILL.md:86]: 73de75abfd46f6b0a98ec1aced02526c063a6b02

File: docs/tr/skills/quarkus-patterns/SKILL.md [hunk 3]
```diff
@@ -0,0 +1,754 @@
...
+name: quarkus-patterns
+description: Quarkus 3.x LTS architecture patterns with Camel for messaging, RESTful API design, CDI services, data access with Panache, and async processing. Use for Java Quarkus backend work with event-driven architectures.
+origin: ECC
+---
+
+# Quarkus Development Patterns
+
+Quarkus 3.x architecture and API patterns for cloud-native, event-driven services with Apache Camel.
+
+## When to Activate
```
[the remark this answers is not in the document]
Comment (Alexsky347) [reply docs/tr/skills/quarkus-patterns/SKILL.md]: b54ce43ef3c77d51f9313324954334397caac1b8

File: docs/zh-CN/skills/quarkus-patterns/SKILL.md [hunk 4]
```diff
@@ -0,0 +1,754 @@
...
+
+# Quarkus Development Patterns
+
+Quarkus 3.x architecture and API patterns for cloud-native, event-driven services with Apache Camel.
+
+## When to Activate
```
[the remark this answers is not in the document]
Comment (Alexsky347) [reply docs/zh-CN/skills/quarkus-patterns/SKILL.md]: b54ce43ef3c77d51f9313324954334397caac1b8

File: docs/ja-JP/skills/quarkus-patterns/SKILL.md [hunk 5]
```diff
@@ -0,0 +1,754 @@
...
+name: quarkus-patterns
+description: Quarkus 3.x LTS architecture patterns with Camel for messaging, RESTful API design, CDI services, data access with Panache, and async processing. Use for Java Quarkus backend work with event-driven architectures.
+origin: ECC
+---
+
+# Quarkus Development Patterns
+
+Quarkus 3.x architecture and API patterns for cloud-native, event-driven services with Apache Camel.
+
+## When to Activate
```
[the remark this answers is not in the document]
Comment (Alexsky347) [reply docs/ja-JP/skills/quarkus-patterns/SKILL.md]: b54ce43ef3c77d51f9313324954334397caac1b8

File: skills/prompt-optimizer/SKILL.md [hunk 6]
```diff
@@ -135,6 +135,7 @@ Map intent + scope + tech stack (from Phase 0) to specific ECC components.
 | Python / Django | django-patterns, django-tdd, django-security, django-verification, python-patterns, python-testing | python-reviewer |
 | Go | golang-patterns, golang-testing | go-reviewer, go-build-resolver |
 | Spring Boot / Java | springboot-patterns, springboot-tdd, springboot-security, springboot-verification, java-coding-standards, jpa-patterns | code-reviewer |
+| Quarkus / Java | quarkus-patterns, quarkus-tdd, quarkus-security, quarkus-verification, java-coding-standards | java-reviewer |
 | Kotlin / Android | kotlin-coroutines-flows, compose-multiplatform-patterns, android-clean-architecture | kotlin-reviewer |
```
[the remark this answers is not in the document]
Comment (Alexsky347) [reply skills/prompt-optimizer/SKILL.md:139]: 63934f382acd378503bb897ff6d0ad668d7591e7

File: docs/ja-JP/skills/quarkus-security/SKILL.md [hunk 7]
```diff
@@ -0,0 +1,453 @@
...
+- Hash passwords with BCrypt (never plaintext)
+- Store secrets in Vault or environment variables
+- Use parameterized queries to prevent SQL injection
+- Add security headers to all responses
+- Implement rate limiting for public endpoints
+- Audit sensitive operations
+- Keep dependencies updated and scan for CVEs
+- Use SecurityIdentity for programmatic checks
+- Set appropriate CORS policies
+- Test authentication and authorization paths
```
[the remark this answers is not in the document]
Comment (Alexsky347) [reply docs/ja-JP/skills/quarkus-security/SKILL.md]: 9b9f71c

File: docs/ja-JP/skills/quarkus-verification/SKILL.md [hunk 8]
```diff
@@ -0,0 +1,481 @@
...
+- Run verification loop before every PR
+- Automate in CI/CD pipeline
+- Fix issues immediately; don't accumulate debt
+- Keep coverage above 80%
+- Update dependencies regularly
+- Test native compilation periodically
+- Monitor performance trends
+- Document breaking changes
+- Review security scan results
+- Validate configuration for each environment
```
[the remark this answers is not in the document]
Comment (Alexsky347) [reply docs/ja-JP/skills/quarkus-verification/SKILL.md]: 9b9f71c

File: docs/zh-CN/skills/quarkus-security/SKILL.md [hunk 9]
```diff
@@ -0,0 +1,453 @@
...
+- Hash passwords with BCrypt (never plaintext)
+- Store secrets in Vault or environment variables
+- Use parameterized queries to prevent SQL injection
+- Add security headers to all responses
+- Implement rate limiting for public endpoints
+- Audit sensitive operations
+- Keep dependencies updated and scan for CVEs
+- Use SecurityIdentity for programmatic checks
+- Set appropriate CORS policies
+- Test authentication and authorization paths
```
[the remark this answers is not in the document]
Comment (Alexsky347) [reply docs/zh-CN/skills/quarkus-security/SKILL.md]: 9b9f71c

File: skills/quarkus-patterns/SKILL.md [hunk 10]
```diff
@@ -0,0 +1,754 @@
...
+- Use `camel-quarkus-junit5` for route testing
+- Use AssertJ for assertions
+- Mock all external dependencies
+- Test conditional flow logic thoroughly
+
+### Quarkus-Specific
+- Stay on latest LTS version (3.x)
+- Use Quarkus dev mode for hot reload
+- Add health checks for production readiness
+- Test native compilation compatibility periodically
```
[the remark this answers is not in the document]
Comment (Alexsky347) [reply skills/quarkus-patterns/SKILL.md:757]: 9b9f71c

File: skills/quarkus-security/SKILL.md [hunk 11]
```diff
@@ -0,0 +1,453 @@
...
+- Hash passwords with BCrypt (never plaintext)
+- Store secrets in Vault or environment variables
+- Use parameterized queries to prevent SQL injection
+- Add security headers to all responses
+- Implement rate limiting for public endpoints
+- Audit sensitive operations
+- Keep dependencies updated and scan for CVEs
+- Use SecurityIdentity for programmatic checks
+- Set appropriate CORS policies
+- Test authentication and authorization paths
```
[the remark this answers is not in the document]
Comment (Alexsky347) [reply skills/quarkus-security/SKILL.md:467]: 9b9f71c

File: skills/quarkus-verification/SKILL.md [hunk 12]
```diff
@@ -0,0 +1,481 @@
...
+- Run verification loop before every PR
+- Automate in CI/CD pipeline
+- Fix issues immediately; don't accumulate debt
+- Keep coverage above 80%
+- Update dependencies regularly
+- Test native compilation periodically
+- Monitor performance trends
+- Document breaking changes
+- Review security scan results
+- Validate configuration for each environment
```
[the remark this answers is not in the document]
Comment (Alexsky347) [reply skills/quarkus-verification/SKILL.md:479]: 9b9f71c

File: docs/ja-JP/skills/quarkus-patterns/SKILL.md [hunk 13]
```diff
@@ -0,0 +1,754 @@
...
+        event.setPayload(serializePayload(payload));
+        event.setTimestamp(Instant.now());
+        
+        eventRepository.persist(event);
+        log.error("Error event created: {} - {}", eventType, errorMessage);
+    }
+    
+    private String serializePayload(Object payload) {
+        // JSON serialization
+        return objectMapper.writeValueAsString(payload);
```
[the remark this answers is not in the document]
Comment (Alexsky347) [reply docs/ja-JP/skills/quarkus-patterns/SKILL.md]: 9b9f71c2e58b65b5d191188de71c80dfaa3571c1

File: docs/ja-JP/skills/quarkus-tdd/SKILL.md [hunk 14]
```diff
@@ -0,0 +1,908 @@
...
+      
+      camelContext.getRouteController().stopRoute("document-processing");
+      AdviceWith.adviceWith(camelContext, "document-processing", advice -> {
+        advice.weaveByToString(".*direct:validation-error-handler.*")
+            .replace().to("mock:error");
+      });
+      camelContext.getRouteController().startRoute("document-processing");
+      
+      // Mock validator to throw exception
+      when(eventService.validate(any())).thenThrow(new ValidationException("Invalid document"));
```
[the remark this answers is not in the document]
Comment (Alexsky347) [reply docs/ja-JP/skills/quarkus-tdd/SKILL.md]: 9b9f71c2e58b65b5d191188de71c80dfaa3571c1

File: docs/ja-JP/skills/quarkus-patterns/SKILL.md [hunk 15]
```diff
@@ -0,0 +1,754 @@
...
+                    String errorMsg = "File path is empty after upload";
+                    log.error(errorMsg);
+                    this.eventService.createErrorEvent(documentInfo, "FILE_UPLOAD_FAILED", errorMsg);
+                    throw new As2ServerProcessingException(errorMsg);
+                }
+                
+                this.eventService.createSuccessEvent(documentInfo, "PERSISTENCE_BLOB_EVENT_TYPE");
+                
+                BusinessRulesPayload payload = this.documentJobService.createDocumentAndJobEntities(
+                    documentInfo, originalFileName, structureIdPartner,
```
[the remark this answers is not in the document]
Comment (Alexsky347) [reply docs/ja-JP/skills/quarkus-patterns/SKILL.md:83]: 9b9f71c2e58b65b5d191188de71c80dfaa3571c1

File: agents/java-build-resolver.md [hunk 16]
```diff
@@ -108,17 +152,51 @@ java -version
...
+
+# Check Spring Boot version alignment
+./mvnw dependency:tree | grep "org.springframework.boot"
+```
+
+## [QUARKUS] Quarkus Specific Commands
+
+```bash
+# Verify Quarkus build augmentation
+./mvnw quarkus:build -q
```
[the remark this answers is not in the document]
Comment (Alexsky347) [reply agents/java-build-resolver.md:177]: e9089cf44e4ba9db9be11249052d35e730aee05b

File: docs/zh-CN/skills/quarkus-security/SKILL.md [hunk 17]
```diff
@@ -0,0 +1,453 @@
...
+    // XSS protection
+    headers.putSingle("X-Content-Type-Options", "nosniff");
+    headers.putSingle("X-XSS-Protection", "1; mode=block");
+    
+    // HSTS
+    headers.putSingle("Strict-Transport-Security", "max-age=31536000; includeSubDomains");
+    
+    // CSP
+    headers.putSingle("Content-Security-Policy", 
+        "default-src 'self'; script-src 'self' 'unsafe-inline'; style-src 'self' 'unsafe-inline'");
```
[the remark this answers is not in the document]
Comment (Alexsky347) [reply docs/zh-CN/skills/quarkus-security/SKILL.md]: 61dfbf8

File: docs/tr/skills/quarkus-tdd/SKILL.md [hunk 18]
```diff
@@ -0,0 +1,908 @@
...
+### Test Coverage
+- Test happy paths for all public methods
+- Test null input handling
+- Test edge cases (empty collections, boundary values, negative IDs, blank strings)
+- Test exception scenarios comprehensively
+- Mock all external dependencies (repositories, services, Camel endpoints)
+- Aim for 80%+ line coverage, 70%+ branch coverage
+
+### Assertions
+- **Always use AssertJ** (`assertThat`) instead of JUnit assertions
```
[the remark this answers is not in the document]
Comment (Alexsky347) [reply docs/tr/skills/quarkus-tdd/SKILL.md]: eb2ea25

File: agents/java-build-resolver.md [hunk 19]
```diff
@@ -138,16 +218,20 @@ Stop and report if:
 - Fix introduces more errors than it resolves
 - Error requires architectural changes beyond scope
 - Missing external dependencies that need user decision (private repos, licences)
+- **[QUARKUS]**: Native image build fails due to GraalVM not being installed — report prerequisite
 
 ## Output Format
 
 ```text
+Framework: [SPRING|QUARKUS|UNKNOWN]
```
[the remark this answers is not in the document]
Comment (Alexsky347) [reply agents/java-build-resolver.md]: 8e5c5f59ce754855919fba18fae2d21dc35cdbd8

File: docs/zh-CN/skills/quarkus-tdd/SKILL.md [hunk 20]
```diff
@@ -0,0 +1,375 @@
...
+    
+    camelContext.getRouteController().stopRoute("business-rules-publisher");
+    AdviceWith.adviceWith(camelContext, "business-rules-publisher", advice -> {
+      advice.replaceFromWith("direct:business-rules-publisher");
+      advice.weaveByToString(".*spring-rabbitmq.*").replace().to("mock:rabbitmq");
+    });
+    camelContext.getRouteController().startRoute("business-rules-publisher");
+    
+    // ACT
+    producerTemplate.sendBody("direct:business-rules-publisher", testPayload);
```
[the remark this answers is not in the document]
Comment (Alexsky347) [reply docs/zh-CN/skills/quarkus-tdd/SKILL.md:212]: 4402e47

File: docs/ja-JP/skills/quarkus-tdd/SKILL.md [hunk 21]
```diff
@@ -0,0 +1,376 @@
...
+    
+    camelContext.getRouteController().stopRoute("business-rules-publisher");
+    AdviceWith.adviceWith(camelContext, "business-rules-publisher", advice -> {
+      advice.replaceFromWith("direct:business-rules-publisher");
+      advice.weaveByToString(".*spring-rabbitmq.*").replace().to("mock:rabbitmq");
+    });
+    camelContext.getRouteController().startRoute("business-rules-publisher");
+    
+    // ACT
+    producerTemplate.sendBody("direct:business-rules-publisher", testPayload);
```
[the remark this answers is not in the document]
Comment (Alexsky347) [reply docs/ja-JP/skills/quarkus-tdd/SKILL.md:212]: 4402e47

File: docs/zh-CN/skills/prompt-optimizer/SKILL.md [hunk 22]
```diff
@@ -116,7 +116,8 @@ metadata:
 |------------|--------------|-------|
```
[the remark this answers is not in the document]
Comment (Alexsky347) [reply docs/zh-CN/skills/prompt-optimizer/SKILL.md:116]: bc94f99

File: docs/zh-CN/skills/quarkus-patterns/SKILL.md [hunk 23]
```diff
@@ -0,0 +1,728 @@
...
+                    log.error(errorMsg);
+                    this.eventService.createErrorEvent(documentInfo, "FILE_UPLOAD_FAILED", errorMsg);
+                    throw new As2ServerProcessingException(errorMsg);
+                }
+                
+                this.eventService.createSuccessEvent(documentInfo, "PERSISTENCE_BLOB_EVENT_TYPE");
+                
+                BusinessRulesPayload payload = this.documentJobService.createDocumentAndJobEntities(
+                    documentInfo, originalFileName, structureIdPartner, 
+                    flowProfile, invoiceValidationResult.getDocumentHash());
```
[the remark this answers is not in the document]
Comment (Alexsky347) [reply docs/zh-CN/skills/quarkus-patterns/SKILL.md:84]: @copilot apply changes based on [this feedback](https://github.com/affaan-m/everything-claude-code/pull/1325#discussion_r3053851748)

File: docs/tr/skills/quarkus-patterns/SKILL.md [hunk 24]
```diff
@@ -0,0 +1,754 @@
...
+                    log.error(errorMsg);
+                    this.eventService.createErrorEvent(documentInfo, "FILE_UPLOAD_FAILED", errorMsg);
+                    throw new As2ServerProcessingException(errorMsg);
+                }
+                
+                this.eventService.createSuccessEvent(documentInfo, "PERSISTENCE_BLOB_EVENT_TYPE");
+                
+                BusinessRulesPayload payload = this.documentJobService.createDocumentAndJobEntities(
+                    documentInfo, originalFileName, structureIdPartner, 
+                    flowProfile, invoiceValidationResult.getDocumentHash());
```
[the remark this answers is not in the document]
Comment (Alexsky347) [reply docs/tr/skills/quarkus-patterns/SKILL.md:84]: @copilot apply changes based on [this feedback](https://github.com/affaan-m/everything-claude-code/pull/1325#discussion_r3053851790)

File: docs/ja-JP/skills/quarkus-patterns/SKILL.md [hunk 25]
```diff
@@ -0,0 +1,754 @@
...
+                    log.error(errorMsg);
+                    this.eventService.createErrorEvent(documentInfo, "FILE_UPLOAD_FAILED", errorMsg);
+                    throw new As2ServerProcessingException(errorMsg);
+                }
+                
+                this.eventService.createSuccessEvent(documentInfo, "PERSISTENCE_BLOB_EVENT_TYPE");
+                
+                BusinessRulesPayload payload = this.documentJobService.createDocumentAndJobEntities(
+                    documentInfo, originalFileName, structureIdPartner, 
+                    flowProfile, invoiceValidationResult.getDocumentHash());
```
[the remark this answers is not in the document]
Comment (Alexsky347) [reply docs/ja-JP/skills/quarkus-patterns/SKILL.md:84]: @copilot apply changes based on [this feedback](https://github.com/affaan-m/everything-claude-code/pull/1325#discussion_r3053851822)

File: docs/zh-CN/skills/configure-ecc/SKILL.md [hunk 26]
```diff
@@ -126,6 +126,10 @@ mkdir -p $TARGET/skills $TARGET/rules
 | `java-coding-standards` | Spring Boot 的 Java 编码标准：命名、不可变性、Optional、流 |
 | `python-patterns` | Pythonic 惯用法、PEP 8、类型提示、最佳实践 |
 | `python-testing` | 使用 pytest、TDD、夹具、模拟、参数化进行 Python 测试 |
```
[the remark this answers is not in the document]
Comment (Alexsky347) [reply docs/zh-CN/skills/configure-ecc/SKILL.md:128]: @copilot apply changes based on [this feedback](https://github.com/affaan-m/everything-claude-code/pull/1325#discussion_r3053851841)

File: docs/ja-JP/skills/configure-ecc/SKILL.md [hunk 27]
```diff
@@ -96,6 +96,10 @@ Options:
 | `java-coding-standards` | Spring Boot 用 Java コーディング標準: 命名、不変性、Optional、ストリーム |
 | `python-patterns` | Pythonic なイディオム、PEP 8、型ヒント、ベストプラクティス |
 | `python-testing` | pytest、TDD、フィクスチャ、モック、パラメータ化による Python テスト |
+| `quarkus-patterns` | Quarkus アーキテクチャ、Camel メッセージング、CDI サービス、Panache データアクセス |
```
[the remark this answers is not in the document]
Comment (Alexsky347) [reply docs/ja-JP/skills/configure-ecc/SKILL.md:99]: @copilot apply changes based on [this feedback](https://github.com/affaan-m/everything-claude-code/pull/1325#discussion_r3053851864)

File: agent.yaml [hunk 28]
```diff
@@ -126,6 +126,10 @@ skills:
   - security-scan
   - skill-comply
   - skill-stocktake
+  - quarkus-patterns
+  - quarkus-security
+  - quarkus-tdd
+  - quarkus-verification
   - springboot-patterns
   - springboot-security
   - springboot-tdd
```
[the remark this answers is not in the document]
Comment (Alexsky347) [reply agent.yaml:135]: @copilot apply changes based on [this feedback](https://github.com/affaan-m/everything-claude-code/pull/1325#discussion_r3053851889)

File: docs/zh-CN/skills/configure-ecc/SKILL.md [hunk 29]
```diff
@@ -126,6 +126,10 @@ mkdir -p $TARGET/skills $TARGET/rules
...
+| `quarkus-patterns` | Quarkus 架构、使用 Camel 的事件驱动模式、Panache 数据访问、CDI 服务 |
+| `quarkus-security` | Quarkus 安全：JWT/OIDC 认证、RBAC、Bean 验证、CORS、密钥管理 |
+| `quarkus-tdd` | 使用 JUnit 5、Mockito、REST Assured、Camel 测试进行 Quarkus TDD |
+| `quarkus-verification` | Quarkus 验证：构建、静态分析、测试、安全扫描、原生编译 |
 | `springboot-patterns` | Spring Boot 架构、REST API、分层服务、缓存、异步处理 |
```
[the remark this answers is not in the document]
Comment (Alexsky347) [reply docs/zh-CN/skills/configure-ecc/SKILL.md:133]: @copilot apply changes based on [this feedback](https://github.com/affaan-m/everything-claude-code/pull/1325#discussion_r3053851908)

File: skills/quarkus-patterns/SKILL.md [hunk 30]
```diff
@@ -0,0 +1,757 @@
...
+            <type>pom</type>
+            <scope>import</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.camel.quarkus</groupId>
+            <artifactId>camel-quarkus-bom</artifactId>
+            <version>${quarkus.platform.version}</version>
+            <type>pom</type>
+            <scope>import</scope>
+        </dependency>
```
[the remark this answers is not in the document]
Comment (Alexsky347) [reply skills/quarkus-patterns/SKILL.md:656]: 893eca03690bc134cda57bc5b7f1d99d88b57028

File: skills/quarkus-security/SKILL.md [hunk 31]
```diff
@@ -0,0 +1,455 @@
...
+              .build()
+      );
+    }
+  }
+
+  private String getClientIdentifier(ContainerRequestContext ctx) {
+    // Use IP, API key, or user ID
+    return ctx.getHeaderString("X-Forwarded-For");
+  }
+}
```
[the remark this answers is not in the document]
Comment (Alexsky347) [reply skills/quarkus-security/SKILL.md]: 8f65048bc3fbe37a176ee6cee152c32bf99d73d0

File: skills/quarkus-security/SKILL.md [hunk 32]
```diff
@@ -0,0 +1,455 @@
...
+    String authHeader = requestContext.getHeaderString(HttpHeaders.AUTHORIZATION);
+    
+    if (authHeader != null && authHeader.startsWith("Bearer ")) {
+      String token = authHeader.substring(7);
+      // Validate token and set SecurityIdentity
+      if (!validateToken(token)) {
+        requestContext.abortWith(Response.status(Response.Status.UNAUTHORIZED).build());
+      }
+    }
+  }
```
[the remark this answers is not in the document]
Comment (Alexsky347) [reply skills/quarkus-security/SKILL.md:86]: eddfeb6

File: skills/quarkus-tdd/SKILL.md [hunk 33]
```diff
@@ -0,0 +1,912 @@
...
+    }
+  }
+
+  @Nested
+  @DisplayName("Tests for createErrorEvent")
+  class CreateErrorEvent {
+    
+    @Test
+    @DisplayName("Should create error event with error message")
+    void givenError_whenCreateErrorEvent_thenEventPersistedWithMessage() throws Exception {
```
[the remark this answers is not in the document]
Comment (Alexsky347) [reply skills/quarkus-tdd/SKILL.md:464]: c399627

File: skills/quarkus-tdd/SKILL.md [hunk 34]
```diff
@@ -0,0 +1,912 @@
...
+
+    @Test
+    @DisplayName("Should throw exception when payload is null")
+    void givenNullPayload_whenCreateSuccessEvent_thenThrowsException() {
+      // ARRANGE
+      Object nullPayload = null;
+      
+      // ACT & ASSERT
+      IllegalArgumentException exception = assertThrows(
+          IllegalArgumentException.class,
```
[the remark this answers is not in the document]
Comment (Alexsky347) [reply skills/quarkus-tdd/SKILL.md]: e928cea

File: skills/quarkus-patterns/SKILL.md [hunk 35]
```diff
@@ -0,0 +1,757 @@
...
+    
+    private String serializePayload(Object payload) {
+        // JSON serialization
+        return objectMapper.writeValueAsString(payload);
+    }
+}
+```
+
+## Camel Message Publishing (RabbitMQ)
+
```
[the remark this answers is not in the document]
Comment (Alexsky347) [reply skills/quarkus-patterns/SKILL.md]: 56bbbb3

File: docs/zh-CN/skills/quarkus-security/SKILL.md (hunk 9 above)
[the remark this answers is not in the document]
Comment (Alexsky347) [reply docs/zh-CN/skills/quarkus-security/SKILL.md]: 9b9f71c

**Comment by Alexsky347** [comment]
@coderabbitai resume

File: skills/quarkus-tdd/SKILL.md [hunk 36]
```diff
@@ -0,0 +1,915 @@
...
+      // ACT
+      producerTemplate.sendBody("direct:process-document", testPayload);
+      
+      // ASSERT
+      mockError.assertIsSatisfied(5000);
+      
+      Exception exception = mockError.getExchanges().get(0).getException();
+      assertThat(exception).isInstanceOf(ValidationException.class);
+      assertThat(exception.getMessage()).contains("Invalid document");
+    }
```
[the remark this answers is not in the document]
Comment (Alexsky347) [reply skills/quarkus-tdd/SKILL.md:388]: 9b9f71c

**Review by Alexsky347 — COMMENTED** [review]
submit the review

**Comment by affaan-m** [comment]
Thanks for the PR. This has been idle for several weeks and is not merge-ready against the moving main branch, so I am closing it to keep the queue tractable. Reopen is welcome with a current rebase and focused scope; maintainers may also port the useful parts into a fresh PR where appropriate.
